src/components/charts/galacticProperties/Point.jsx

```diff
@@ -1,59 +1,17 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { select as d3Select } from 'd3-selection';
-import { easeElastic as d3EaseElastic } from 'd3-ease';
+import SvgShape from '../shared/shapes';
 
 class Point extends React.PureComponent {
   constructor(props) {
     super(props);
 
-    this.baseRadius = 6;
-    this.svgEl = React.createRef();
-  }
-
-  componentDidUpdate() {
-    const { selected, hovered } = this.props;
-    const $point = d3Select(this.svgEl.current);
-    if (selected || hovered) {
-      $point
-        .transition()
-        .duration(800)
-        .ease(d3EaseElastic)
-        .attr('r', this.baseRadius * 2);
-    } else {
-      $point
-        .transition()
-        .duration(400)
-        .attr('r', this.baseRadius);
-    }
+    this.height = 15;
+    this.width = 15;
```

Comment:
As noted elsewhere, it seems like you're always adjusting the actual height and width used in each shape.  If that must be the case, then maybe no value in passing these hardcoded values down?  Or at the very least, since all our shapes are symmetrical (...even?..."fit in a square") maybe just have one value?  And since only yused in one place can be declared in prop it's assigned to rather than as a class variable.

Reply:
Will use a `const dimension = 15` and apply that to the props as discussed. Will also use the `import Point from '../svgShape';` approach as well.